mobilecoind/src/sync.rs

```diff
@@ -355,7 +354,7 @@ fn match_tx_outs_into_utxos(
         let tx_public_key = RistrettoPublic::try_from(&tx_out.public_key)?;
 
         let subaddress_spk = SubaddressSPKId::from(&recover_public_subaddress_spend_key(
-            &view_key.view_private_key,
+            account_key.view_private_key(),
```

Comment:
this change here is the same change that I think full service will have to make

Reply:
Here is a PR for it https://github.com/mobilecoinofficial/full-service/pull/271